src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -112,12 +114,11 @@ public ActionPlanDTO createActionPlan(
     final MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
     queryParams.add(SELECTOR_COLLECTION_EXERCISE_ID, collectionExerciseId);
     queryParams.add(SELECTOR_ACTIVE_ENROLMENT, Boolean.toString(activeEnrolment));
-
-    HttpEntity httpEntity = restUtility.createHttpEntity(null);
-
     final UriComponents uriComponents =
         restUtility.createUriComponents(appConfig.getActionSvc().getActionPlansPath(), queryParams);
 
+    HttpEntity httpEntity = restUtility.createHttpEntity(null);
```

Comment:
Not sure sure about passing null to the rest utility.

Reply:
@atrozzo @andrewmil  Check out https://github.com/ONSdigital/rm-common-service/pull/14

Reply:
I'll use that solution once it's merged :+1: